### ISSUE TITLE
Rename master → main

### ISSUE THREAD

**Issue by alecjacobson** [libigl/libigl-python-bindings#156]
We should do this. This issue is a heads up and to find out if there's going to be anything to change w.r.t. the legacy conda setup.

Local clones will have to do:
```
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```

**Comment by alecjacobson** [comment]
Should do this as soon as https://github.com/libigl/libigl-python-bindings/pull/164 is merged.

**Comment by alecjacobson** [comment]
Fixed by fd02f80fff4cd9e127b61149fc8d645960a9b569
